### PR TITLE
run test cases in travis

### PR DESCRIPTION
**Pull request by sendilkumarn**
How to enable the build ?

### CODE REVIEW

**Comment by jdubois** [comment]
You need to ask me :-)

**Comment by jdubois** [comment]
It should be configured now, it will start with your next push at https://travis-ci.org/jhipster/ng-jhipster

**Comment by sendilkumarn** [comment]
@jdubois Oh yeah, Please do the needful 👍

File: .travis.yml [hunk 1]
```diff
@@ -30,4 +30,7 @@ install:
 
 before_script:
   - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
\ No newline at end of file
+  - sh -e /etc/init.d/xvfb start
+
+script: 
+  - npm run test
```
Comment (jdubois) [inline .travis.yml]: you're still using NPM, not Yarn?

**Comment by pascalgrimaud** [comment]
@jdubois : can you activate this project in Travis plz ?

But I see a problem: we begin to have a lot of projects under our organization.
Maybe we can use other CI ? In the past, there was drone.io : https://drone.io/github.com/jhipster/jhipster-sample-app/latest with `this build successfully completed 10 days ago`

It's still used !!!

File: .travis.yml (hunk 1 above)
Comment (sendilkumarn) [inline .travis.yml]: my bad changed.

**Comment by pascalgrimaud** [comment]
I'm too slow, it's just activated!

**Comment by sendilkumarn** [comment]
@pascalgrimaud we can try out something new 👍

**Comment by jdubois** [comment]
That was easy :-)
Yes we do have a lot of builds, but only the generator is slow, and thanks to Yarn I'm not too worried. 2 CI systems means 2 times more work, we have better things to do...

**Comment by pascalgrimaud** [comment]
Travis shares all the instances inside the same organization, that means, if there are a lot of work on generator-jhipster, all other projects need to wait.
But we can discuss about this next year, after JHipster 4 :-) it's not urgent

**Comment by jdubois** [comment]
Yes I know... But we have 5 boxes, it's already quite a lot, and it doesn't take that much time...

**Comment by pascalgrimaud** [comment]
Don't worry, we'll see it later ;)
I know, it's not the right place to discuss about it, but Yarn uses AppVeyor, a CI for Windows, using PowerShell: https://ci.appveyor.com/project/kittens/yarn/build/job/gt2lnwu63m3dk680
Ping @deepu105 : I'm sure you'll like it! :heart:

**Comment by deepu105** [comment]
@pascalgrimaud nope I moved to Linux (Still miss my windows though :( )

**Comment by deepu105** [comment]
@jhipster/developers a piece of advice/info regarding Yarn.

So I was happily coding this project and using yarn everywhere so once it was kind of ready I wanted to test it before publishing so I did
1. `yarn link` on this project -> it registered succesfully
2. created a new jhipster app and refactored it to use this lib
3. did `yarn link ng-jhipster` on the new JH app
4. In vscode everything seemed perfect I was able to import from `ng-jhipster` and code seemed to compile. No errors or anything
5. Now I started testing and it wasnt working there was weird errors from angular core and I had no idea why. i started to debug in vain. tried different combinations to bootstrap. tried lazy loading etc etc.
6. I short I wasted almost 2 days trying to figure out what was wrong
7. Then it occured to me 'lets try once with npm'
8. I cleared all node modules
9. did `npm install` everywhere and used `npm link` and `npm link ng-jhipster`
10. Voila it worked with my original code without any issue

So i'm not sure if it was something wrong with how yarn link works or with how webpack handles linked files via yarn but it was too troublesome and there was no clue at all. It could also be some issue with my setup.

So incase you happen to encounter similar issue you know what to do now :)

**Comment by pascalgrimaud** [comment]
@deepu105 : I will have a look on it, as I played a lot with yarn recently

**Comment by jdubois** [comment]
Oh I had no idea you had such issues @deepu105 !!!
For me Yarn works well now, but indeed I sometimes have some weird stuff happening...

**Comment by pascalgrimaud** [comment]
@deepu105 : ok, I spend a couple of hours on this, and indeed, I have the same issues than you.
It's clearly something related to symlink...

**Comment by deepu105** [comment]
For me it works when done with NPM I have the issue only with Yarn

Thanks & regards,
Deepu

**Pull request merged**